src/Microsoft.OData.Edm/Csdl/Semantics/CsdlSemanticsTypeDefinitionDefinition.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.OData.Edm.Csdl.CsdlSemantics
     /// <summary>
     /// Provides semantics for CsdlTypeDefinition.
     /// </summary>
-    internal class CsdlSemanticsTypeDefinitionDefinition : CsdlSemanticsTypeDefinition, IEdmTypeDefinition, IEdmFullNamedElement
+    internal class CsdlSemanticsTypeDefinitionDefinition : CsdlSemanticsTypeDefinition, IEdmTypeDefinition, IEdmFullNamedElement, IEdmFacetedTypeDefinition

```

Comment:
Since `IEdmFacetedTypeDefinition` implements `IEdmTypeDefinition`, do we need `IEdmTypeDefinition` here?

Reply:
@KenitoInc I'd be more comfortable leaving it as is since there's no harm. If we merge `IEdmFacetedTypeDefinition` into `IEdmTypeDefinition` in the next major release, the action will be seamless.